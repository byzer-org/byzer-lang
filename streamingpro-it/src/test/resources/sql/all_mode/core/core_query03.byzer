load parquet.`/ssb/customer/part-00000-7505a64d-b4bf-4186-874a-a9898502a51e-c000.snappy.parquet` as customer_parquet;

select * from customer_parquet as customer;

select split(C_NAME,"#") as C_NAME_SPLIT from customer as customer_split;

select
    C_NAME_SPLIT[0] as C_NAME_1,
    C_NAME_SPLIT[1] as C_NAME_2
from customer_split
limit 100
as customer_split_format;
