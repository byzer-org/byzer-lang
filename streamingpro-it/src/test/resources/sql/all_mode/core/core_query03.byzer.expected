C_NAME_1,C_NAME_2
Customer,000000001
Customer,000000002
Customer,000000003
Customer,000000004
Customer,000000005
Customer,000000006
Customer,000000007
Customer,000000008
Customer,000000009
Customer,000000010
Customer,000000011
Customer,000000012
Customer,000000013
Customer,000000014
Customer,000000015
Customer,000000016
Customer,000000017
Customer,000000018
Customer,000000019
Customer,000000020
Customer,000000021
Customer,000000022
Customer,000000023
Customer,000000024
Customer,000000025
Customer,000000026
Customer,000000027
Customer,000000028
Customer,000000029
Customer,000000030
Customer,000000031
Customer,000000032
Customer,000000033
Customer,000000034
Customer,000000035
Customer,000000036
Customer,000000037
Customer,000000038
Customer,000000039
Customer,000000040
Customer,000000041
Customer,000000042
Customer,000000043
Customer,000000044
Customer,000000045
Customer,000000046
Customer,000000047
Customer,000000048
Customer,000000049
Customer,000000050
Customer,000000051
Customer,000000052
Customer,000000053
Customer,000000054
Customer,000000055
Customer,000000056
Customer,000000057
Customer,000000058
Customer,000000059
Customer,000000060
Customer,000000061
Customer,000000062
Customer,000000063
Customer,000000064
Customer,000000065
Customer,000000066
Customer,000000067
Customer,000000068
Customer,000000069
Customer,000000070
Customer,000000071
Customer,000000072
Customer,000000073
Customer,000000074
Customer,000000075
Customer,000000076
Customer,000000077
Customer,000000078
Customer,000000079
Customer,000000080
Customer,000000081
Customer,000000082
Customer,000000083
Customer,000000084
Customer,000000085
Customer,000000086
Customer,000000087
Customer,000000088
Customer,000000089
Customer,000000090
Customer,000000091
Customer,000000092
Customer,000000093
Customer,000000094
Customer,000000095
Customer,000000096
Customer,000000097
Customer,000000098
Customer,000000099
Customer,000000100